Lib/test/test_nntplib.py

```diff
@@ -73,6 +73,8 @@ def test_newgroups(self):
             self.assertIsInstance(groups[0], GroupInfo)
             self.assertIsInstance(groups[0].group, str)
 
+    @unittest.skipIf(True, 'Broken skip until upstream fix is applied'
+                           ' see https://bugs.python.org/issue42794')
```

Comment:
These should probably be dropped since we'll end up rebasing after this is merged.

Reply:
Ok